frontend/src/actions/OpenInNextcloud.js

```diff
@@ -22,17 +22,19 @@ class OpenInNextcloud extends Action {
     execute (pointerEvent) {
         console.log(pointerEvent);
         let selectedFiles = store.getters.StateSelectedChildren;
-        let currentDir = store.getters.StatePath[store.getters.StatePath.length-1].name;
+        let currentDirPath = store.getters.StatePath[store.getters.StatePath.length-1].path;
+        let currentDirPathSuffix = currentDirPath.replace(new RegExp('.*' + "com"), '');
```

Comment:
it would be a bit cleaner to make the regex expression a variable that describe what it is supposed to match

something like also you could write a regex directly with `/.*com/` for example

```suggestion
        let matchUntilDotComRegex = /.*com/;
        let currentDirPathSuffix = currentDirPath.replace(matchUntilDotComRegex, '');
```